.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
```

Comment:
This file replaces the travis config, since travis builds weren't running anymore.